proj.py

```diff
@@ -0,0 +1,86 @@
+import numpy as np # linear algebra
```

Comment:
Your imports should be sorted alphabetically.
Read more about imports in pep-8 coding standard style guide: https://www.python.org/dev/peps/pep-0008/#imports
Pro tip: You can use isort package of python which automatically sort imports for python files.

Reply:
Imports sorted alphabetically using isort and updated the file in accordance with pep-8 coding standard.